io.github.fheroes2.yaml

```diff
@@ -0,0 +1,31 @@
+app-id: io.github.fheroes2
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+finish-args:
+  - --filesystem=home
+  - --share=ipc
+  - --socket=wayland
```

Comment:
does it really support wayland?

also doesn't it require openGL?

Reply:
I fixed it